controller_manager/src/controller_manager_parameters.yaml

```diff
@@ -134,3 +134,74 @@ controller_manager:
                 gt<>: 0.0,
               }
             }
+      hardware_components:
+        periodicity:
+          mean_error:
+            warn: {
+              type: double,
+              default_value: 5.0,
+              description: "The warning threshold for the mean error of the hardware component's read/write loop. If the mean error exceeds this threshold, a warning diagnostic will be published. This diagnostics will be published only for the asynchronous hardware component, because any affect to the synchronous hardware components will be reflected directly in the controller manager's periodicity.",
+              validation: {
+                gt<>: 0.0,
+              }
+            }
+            error: {
+              type: double,
+              default_value: 10.0,
+              description: "The error threshold for the mean error of the hardware component's read/write loop. If the mean error exceeds this threshold, an error diagnostic will be published. This diagnostics will be published only for the asynchronous hardware component, because any affect to the synchronous hardware components will be reflected directly in the controller manager's periodicity.",
```

Comment:
can we move some parts of the description to the parameters_context.yaml file, because they are repeating a lot? but I see that it is not sorted correctly in the [rst](https://control.ros.org/rolling/doc/ros2_control/controller_manager/doc/userdoc.html#parameters), I'll have a look to fix that.

Reply:
I agree with you. Let's get this in as it is consistent with others for now, and let's fix it in a different PR?

Reply:
the sorting is fixed now with https://github.com/ros-controls/control.ros.org/pull/384

Reply:
Awesome. I fixed the paramters_context.yaml as you suggested :)